coil-gif/README.md

```diff
@@ -0,0 +1,27 @@
+# coil-gif
+
+Adds GIF decoding support to Coil.
+
+```kotlin
+implementation("io.coil-kt:coil-gif:latest.version.here")
```

Comment:
Let's hardcode the latest version. We can find/replace during the deploy process.